### PR TITLE
Cautionary contacts again

### PR DESCRIPTION
**Pull request by pudiva**

### CODE REVIEW

File: app/assets/stylesheets/hackney_work_order.scss [hunk 1]
```diff
@@ -46,7 +46,8 @@
 .main-section-box {
   min-height: 252px;
 }
-.hackney-main-section-info p, ul {
+.hackney-main-section-info p,
+.hackney-main-section-info ul {
```
Comment (cdccollins) [inline app/assets/stylesheets/hackney_work_order.scss:50]: why do it this way?

Comment (pudiva) [reply app/assets/stylesheets/hackney_work_order.scss:50]: The line I removed was matching all `p` inside a `.hackney-main-section-info` and every `ul` (regardless of being inside `.hackney-main-section-info`).

I just prepended the class to the `ul`, which I suppose was the intended effect, to fix it interfering with the new warning labels I did.

File: app/views/properties/_warning_label.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body-s hackney-property-warning-label hackney-property-warning-label-<%= colour %>">
```
Comment (cdccollins) [inline app/views/properties/_warning_label.html.erb]: I can't find where this partial is used in the code

File: app/assets/stylesheets/hackney_work_order.scss (hunk 1 above)
Comment (pudiva) [reply app/assets/stylesheets/hackney_work_order.scss:50]: *warning labels for tenure type, tmo and cautionary contact

File: app/views/properties/_warning_label.html.erb (hunk 2 above)
Comment (pudiva) [reply app/views/properties/_warning_label.html.erb]: This was a mistake. Removing.

Comment (pudiva) [reply app/views/properties/_warning_label.html.erb]: Done. Also removed another stray template.

Comment (cdccollins) [reply app/views/properties/_warning_label.html.erb]: Hahah yeah was just about to comment on that one

**Review by cdccollins — APPROVED** [review]

**Pull request merged**